Source/WebCore/css/CSSFontFeatureValue.cpp

```diff
@@ -45,15 +45,15 @@ String CSSFontFeatureValue::customCSSText() const
     for (char c : m_tag)
         builder.append(c);
     builder.append('"');
-    // Omit the value if it's 1 as 1 is implied by default.
-    if (m_value != 1)
-        builder.append(' ', m_value);
+    // Omit the value if it's `1` as `1` is implied by default.
+    if (m_value->isCalculated() || m_value->resolveIntegerNoConversionDataRequired() != 1)
```

Comment:
Should we have something that returns an `optional` instead of using `isCalculated` as a preflight? It’s not obvious that `isCalculated` and `NoConversionDataRequired` are a perfect match. Doesn’t seem that unusual to want a value excluding the calculated one.